Makefile

```diff
@@ -83,7 +83,6 @@ BPF_MESSAGE := "without BPF support"
 with_bpf := no
 ifeq ("$(OS)","linux")
 ifeq ("$(ARCH)","amd64")
-ifeq ("$(FIPS)","")
```

Comment:
Does this mean that we will now always build FIPS binary on linux/amd64 if libbpf is installed, instead of requesting FIPS build explicitly?

IMO "explicit is better than implicit" so changing the type of build depending on the presence of a library might be a tad confusing. Is this the behavior we want?

Reply:
Good idea, I like it.

Do you mind if I do this in another PR? I made the changes to OSS already, but I have to make changes to `e` as well. Want to cut `7.0.0-beta.2` today and don't want to go through another round of `e` + OSS changes.

Reply:
Sure sounds good to me.